EMULATOR/EMULATOR.C

```diff
@@ -237,19 +237,21 @@ int Emulator_InitialiseGameVariables()
 	extern unsigned long* GadwOrderingTables_V2;
 	extern unsigned long* terminator;
 
+// MAGICAL CONST 231604 is equal to (5128 * 4) + (52260 * 4) + (512 * 4) + 4 WHY?
 #if _WINDOWS
 	SYSTEM_INFO info;
 	GetSystemInfo(&info);
-	Emulator_AllocateVirtualMemory((unsigned int)info.lpMinimumApplicationAddress, (5128 * 4) + (52260 * 4) + (512 * 4) + 4);
+	Emulator_AllocateVirtualMemory((unsigned int)info.lpMinimumApplicationAddress, 231604);
 #else
-	Emulator_AllocateVirtualMemory(0x400000, (5128 * 4) + (52260 * 4) + (512 * 4) + 4);
+	// Interested to know why this does this. Where were these magical numbers divined from?
+	Emulator_AllocateVirtualMemory(0x400000, 231604);
```

Comment:
Also isn't this the DLL minimum entrypoint? Is there really no constant for that?

Source: https://docs.microsoft.com/en-us/previous-versions/ms809762(v=msdn.10)

Reply:
According to this:
https://devblogs.microsoft.com/oldnewthing/20141003-00/?p=43923

It is minimum base address of an EXE file, not DLL.